.github/policies/moderatorTriggers.yml

```diff
@@ -222,14 +222,22 @@ configuration:
                 then:
                   - addLabel:
                       label: Issue-Feature
-              # Highest-Version-Remaining
+              # Highest-Version-Removal
               - if:
                   - commentContains:
-                      pattern: '\[[Pp]olicy\]\s+[Hh]ighest[\s-][Vv]ersion([\s-][Rr]emaining)?'
+                      pattern: '\[[Pp]olicy\]\s+[Hh]ighest[\s-][Vv]ersion([\s-][Rr]emoval)?'
```

Comment:
Can you move this to be sorted alphabetically by label name? It helps when comparing with the list of available labels